modules/ossm-kiali-viewing-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+[id="ossm-viewing-metrics_{context}"]
+= Viewing metrics in the Kiali console
+
+You can view inbound and outbound metrics for your applications, workloads, and services in the Kiali console.  The Detail pages include the following tabs:
+
+* inbound Application metrics
+* outbound Application metrics
+* inbound Workload metrics
+* outbound Workload metrics
+* inbound Service metrics
```

Comment:
Are these GUI items? as in, can you click them? If so, they might need ot be in asterisks, as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting.

Reply:
The actual tabs are **Inbound Metrics** and **Outbound Metrics** on the Application Detail pages and **Inbound Metrics** and **Outbound Metrics** on the Workload Detail pages, etc.  so I think this is OK, since it's not the exact GUI text.